README.md

```diff
@@ -139,8 +139,6 @@ In this workflow, annoFuse performs standardization of StarFusion and arriba out
 
 Please refer to [annoFuse](https://github.com/d3b-center/annoFuse) R package for additional applications like putative oncogene annotations.
 
-Current version is 0.90.0, docker pull tag: `kfdrc/annofuse:0.90.0`
```

Comment:
Hmmm, you changed to the new location for somatic wf, but just removed it here?

Reply:
Felt more out of place here. We don't have a catalog of tools here like we do in the somatic workflow readme. I could go either way on it. It's nice to have the versions plainly written out in the README, but it also risks it being out of sync with the reality in the tools.